addons/nuxeo-retention/nuxeo-retention-web-ui/src/main/resources/web/nuxeo.war/ui/document/retentionrule/nuxeo-retentionrule-view-layout.html

```diff
@@ -0,0 +1,127 @@
+<!--
+`nuxeo-retentionrule-view-layout`
+@group Nuxeo UI
+@element nuxeo-retentionrule-view-layout
+-->
+<dom-module id="nuxeo-retentionrule-view-layout">
+  <template>
+    <style include="iron-flex iron-flex-alignment nuxeo-styles">
+      *[role=widget] {
+        margin-bottom: 16px;
+      }
+    </style>
+
+    <retention-config-widget document="[[document]]"></retention-config-widget>
+
+    <nuxeo-card heading="[[i18n('retention.rule.label.heading')]]">
+      <div role="widget">
+        <label>[[i18n('retention.rule.label.ruleName')]]</label>
+        <div name="title">[[document.properties.dc:title]]</div>
+      </div>
+
+      <div role="widget" hidden$="[[!document.properties.dc:description]]">
+        <label>[[i18n('retention.rule.label.ruleDescription')]]</label>
+        <div name="description" class="multiline">[[document.properties.dc:description]]</div>
+      </div>
+
+      <div role="widget">
+        <label hidden$="[[!document.properties.retention_rule:enabled]]">[[i18n('retention.rule.label.enabled')]]</label>
+        <label hidden$="[[document.properties.retention_rule:enabled]]">[[i18n('retention.rule.label.disabled')]]</label>
```

Comment:
Assuming that there is always a label (i.e., `enabled` is a boolean flag), probably we could stamp only one label and compute the label according the flag.